_meta/ecs-migration.yml

```diff
@@ -35,42 +38,42 @@
 - from: context.system.platform
   to: host.os.platform
 
-#- from: context.request.method
-#  to: http.request.method
-#
-#- from: context.request.http_version
-#  to: http.version
-#
-#- from: context.request.body
-#  to: http.request.body
-#  index: false
-#
-## Not in ECS
-#- from: context.request.cookies
-#  to: http.request.headers.cookies.parsed
-#  index: false
-#
-## Not in ECS
-#- from: context.request.headers.cookie
-#  to: http.request.headers.cookies.original
-#  index: false
-#
-#- from: context.request.headers.user-agent
-#  to: http.request.headers.user-agent
-#  index: false
-#
-#- from: context.request.headers.content-type
-#  to: http.request.headers.content-type
-#  index: false
-#
-#- from: context.request.env
-#  to: http.request.env
-#  index: false
-#
-#- from: context.request.socket
-#  to: http.request.socket
-#  index: false
-#
+- from: context.request.method
+  to: http.request.method
+
+- from: context.request.http_version
+  to: http.version
+
+- from: context.request.body
+  to: http.request.body.content
+  index: false
+
+# Not in ECS
+- from: context.request.cookies
+  to: http.request.headers.cookies.parsed
+  index: false
+
+# Not in ECS
+- from: context.request.headers.cookie
+  to: http.request.headers.cookies.original
+  index: false
+
+- from: context.request.headers.user-agent
+  to: http.request.headers.user-agent
+  index: false
+
+- from: context.request.headers.content-type
+  to: http.request.headers.content-type
+  index: false
+
+- from: context.request.env
+  to: http.request.env
+  index: false
```

Comment:
not defined in ECS

Reply:
The standard in ECS is that if the name does not end with `text` then it is a `keyword`. This is an object, so not sure how to bring this into ECS convention.

Reply:
@graphaelli do you have any opinion on this?

Reply:
The recommendation is to have all fields as `keyword`. If you need `text` datatype, you should have it as a multi-field. This is the reverse of the ES default.

Since this field is not in ECS, you're free to do what you want here.

Reply:
And about things that aren't in ECS (like `http.request.env`), they're not automatically a problem. If we don't add it to ECS, there will never be a conflict. And in this case, it's perfectly fine for you to define this field there.

`http.request.env` is the variables defined in the application's context? If that's the case, I can guarantee it'll never be in ECS (as it has nothing to do with the HTTP protocol), and would not named like this (as we try to use full words as much as possible).

So no conflict here 👍